cloudinit/sources/DataSourceOVF.py

```diff
@@ -440,6 +453,8 @@ def get_max_wait_from_cfg(cfg):
 def wait_for_imc_cfg_file(filename, maxwait=180, naplen=5,
                           dirpath="/var/run/vmware-imc"):
     waited = 0
+    if maxwait <= naplen:
+        naplen = 1
```

Comment:
Nit: would `naplen = maxwait` be better here?

Reply:
If user set maxwait <= naplen,  it seems he is sensitive to this time delay, so check the cfg file at each seconds might be better. How do you think?

Reply:
@TheRealFalcon

Reply:
Sure, that makes sense.